contributors/design-proposals/cpu-manager.md

```diff
@@ -134,12 +134,24 @@ type CPUSet map[int]struct{} // set operations and parsing/formatting helpers
 type CPUTopology TBD
 ```
 
+#### Configuring the CPU Manager
+
 Kubernetes will ship with three CPU manager policies. Only one policy is
 active at a time on a given node, chosen by the operator via Kubelet
 configuration. The three policies are **no-op**, **static** and **dynamic**.
 
-Operators can set the active CPU manager policy through a new Kubelet
-configuration setting `--cpu-manager-policy`.
+The active CPU manager policy is set through a new Kubelet
+configuration value `--cpu-manager-policy`.
+
+The number of CPUs that pods may run on is set using the existing
+node-allocatable configuration settings. See the [node allocatable proposal
```

Comment:
Use this link https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node-allocatable.md#phase-2---enforce-allocatable-on-pods for node-allocatable.